artifactory_cleanup/cli.py

```diff
@@ -130,11 +147,27 @@ def main(self):
                 size(summary.artifacts_size),
             ]
             table.add_row(row)
+            output_json["policies"].append(
+                {
+                    "name": summary.policy_name,
+                    "file_count": summary.artifacts_removed,
+                    "size": size(summary.artifacts_size),
+                }
+            )
 
         table.add_row(["", "", ""])
         table.add_row(["Total size: {}".format(size(total_size)), "", ""])
         print(table)
 
+        output_json["total_size"] = size(total_size)
+
+        if self._output_file:
+            with open(self._output_file, "w") as file:
+                if self._output_format == "table":
+                    file.write(table.get_string())
+                else:
+                    file.write(json.dumps(output_json, indent=4))
```

Comment:
The best practice is to create json without indent because space symbols can be pretty big.
Anyway the file is supposed to be machine readable, so no needs for indents here